server/controllers/finance/feeCenter.js

```diff
@@ -0,0 +1,229 @@
+/**
+* Fee Center Controller
+*
+* This controller exposes an API to the client for reading and writing Fee Center
+*/
+
+const q = require('q');
+const db = require('../../lib/db');
+const NotFound = require('../../lib/errors/NotFound');
+
+// GET /fee_center
+function lookupFeeCenter(id) {
+
+  const sqlFeeCenter = `
+    SELECT id, label, is_principal FROM fee_center WHERE id = ?`;
+
+  const sqlReferenceFeeCenter = `
+    SELECT id, fee_center_id, account_reference_id, is_cost 
+    FROM reference_fee_center 
+    WHERE fee_center_id = ?`;
+
+  const sqlServicesFeeCenter = `
+    SELECT service_fee_center.fee_center_id, service_fee_center.service_id AS id, service.name
+    FROM service_fee_center
+    JOIN service ON service.id = service_fee_center.service_id
+    WHERE fee_center_id = ?`;
+
+  return q.all([
+    db.exec(sqlFeeCenter, [id]),
+    db.exec(sqlReferenceFeeCenter, [id]),
+    db.exec(sqlServicesFeeCenter, [id]),
+  ])
+    .spread((feeCenter, references, services) => {
+      const data = {
+        feeCenter,
+        references,
+        services,
+      };
+
+      return data;
+    });
+}
+
+// Lists
+function list(req, res, next) {
+  const sql = `
+    SELECT f.id, f.label, f.is_principal, GROUP_CONCAT(' ', LOWER(ar.abbr)) AS abbrs, 
```

Comment:
Wow, I had never heard of `GROUP_CONCAT` before!  Awesome - this is really good to know about.